### DISCUSSION TITLE
laravel port is changing automatically ?

### DISCUSSION THREAD

**Discussion by Ensaia** [laravel/framework#35733] [Q&A]
https://user-images.githubusercontent.com/19869364/103215737-ea13a500-491c-11eb-98d2-5b50dd231880.mp4

- Laravel Version: 8.11.2
- PHP Version: 9.4.9
- Database Driver & Version: MySQL 10.4.14-MariaDB

- @vue/cli 4.5.9

### Description:
laravel port is changing automatically after a while(8000=>8001....) or when i refresh the page or navigate to another### Steps To Reproduce:

1. i try to add the option --port but the server stopped running

**Comment by sikhlana** [comment]
The PHP development server is crashing for some reason. Have you checked the error logs?

**Reply by Ensaia** [reply]
it's said
found it in /var/log/php7.4-fpm.log
```
fpm is running, pid 878
ready to handle connections
systemd monitor interval set to 10000ms
Terminating ...
exiting, bye-bye!
```

**Reply by vedprakash4950** [reply]
Hey @Ensaia90  Did you get solution for this please , I am having same problem.

**Reply by Umid-Berdiev** [reply]
Hello, @Ensaia90. Did you find solution for this issue? I've got the same problem.

**Comment by Kamandlou** [comment]
PHP does not have version 9.4.9.

**Comment by intuji-vijayrana** [comment]
One of the reasons that the port is changing automatically or stopped is because of the infinite loop inside the code. Suppose you have instantiated the self class on the constructor or may have looped through the class in the constructor.

**Comment by hussein4alaa** [comment]
too many reasons to change port automatically,
infinite loop inside called function or some errors make your app crashed.
If you can upload your own function that stops the application after requesting it I can help you

**Comment by StrongLucky** [comment]
Just had the same problem. On my server it worked - localy not.

My solution was to replace the PHP-Version on my pc. Maybee this can help some developers and save some time ;)

**Reply by henzeb** [reply]
I really hope the server is a dev server and not a production server or in case of the latter: it uses nginx or apache or similar application to serve your website..
